README.md

```diff
@@ -585,14 +606,17 @@ Keep in mind, that all updates to the NVML bindings code should be made in the
 code be processed and pushed to the `pkg/` directory for release.
 
 ## Releasing
+
```

Comment:
why the newline?

Reply:
`markdownlint` suggests always having a linebreak after titile.